Code/GraphMol/Wrap/testSubstructureMatch.py

```diff
@@ -0,0 +1,118 @@
+#
+#  Copyright (C) 2025 Greg Landrum
+#         All Rights Reserved
+#
+#  This file is part of the RDKit.
+#  The contents are covered by the terms of the BSD license
+#  which is included in the file license.txt, found at the root
+#  of the RDKit source tree.
+#
+
+#
+import unittest
+
+from rdkit import Chem
+
+
+class TestCase(unittest.TestCase):
```

Comment:
Since we're now using `pytest` to run Python tests, should we start using `pytest` syntax, instead of `unittest` ? In my experience, `pytest` is more powerful when it comes to writing parametrized tests and test fixtures, and, sadly, some of these features are not compatible with `unittest`.

Reply:
Well, we're already using `pytest` to run the tests...